tests/postgres_tests/test_aggregates.py

```diff
@@ -151,12 +145,21 @@ def test_default_argument(self):
                     )
                     self.assertEqual(values, {"aggregation": expected_result})
 
+    def test_ordering_warns_of_deprecation(self):
+        msg = "The ordering argument is deprecated. Use order_by instead."
+        with self.assertWarnsMessage(RemovedInDjango60Warning, msg):
+            values = AggregateTestModel.objects.aggregate(
+                arrayagg=ArrayAgg("integer_field", ordering=F("integer_field").desc())
+            )
+            self.assertEqual(values, {"arrayagg": [2, 1, 0, 0]})
```

Comment:
The following should ensure that the proper `stacklevel` is passed.

```suggestion
        with self.assertWarnsMessage(RemovedInDjango60Warning, msg) ctx:
            values = AggregateTestModel.objects.aggregate(
                arrayagg=ArrayAgg("integer_field", ordering=F("integer_field").desc())
            )
            self.assertEqual(values, {"arrayagg": [2, 1, 0, 0]})
        self.assertEqual(ctx.filename, __file__)
```